src/__tests__/Project.test.js

```diff
@@ -156,16 +156,16 @@ describe('Project', () => {
     let project = await Project.init(f.find('nested-workspaces'));
     let packages = await project.getPackages();
     let filtered = await project.filterPackages(packages, {
-      only: 'foo'
+      only: ['foo', 'bar']
```

Comment:
In updating these tests you're also changing the code paths they used to test. 

Would it be possible to keep them as close to the old tests as possible and add new tests for the new functionality?

i.e

```
let filtered = await project.filterPackages(packages, {
-  only: 'foo'
+  only: ['foo']
```

That way we know the base cases still work?